src/patternfly/components/Progress/progress.scss

```diff
@@ -84,7 +90,7 @@
   &.pf-m-success { // set color variables to the success colors
     --pf-c-progress__bar--BackgroundColor: var(--pf-c-progress--m-success--BackgroundColor);
     --pf-c-progress__status--Color: var(--pf-c-progress__status--m-success--Color);
-    --pf-c-progress__measure--inside--Color: var(--pf-c-progress__measure--inside-success--Color);
+    --pf-c-progress--m-inside__measure--Color: var(--pf-c-progress__measure--m-inside--m-success--Color);
```

Comment:
@mcoker should we also change the assignment on this line to match the updated variable name `var(--pf-c-progress__measure--m-inside--m-success--Color);` . 
would it be: `var(--pf-c-progress--m-success__measure--Color);` OR
`var(--pf-c-progress__measure--m-success--Color);` like the variables above

Reply:
oh good catch. It's hard to type out so I just pulled down the branch and created a PR - https://github.com/christiemolloy/patternfly-next/pull/6

Let me know if that looks OK. I think that better grouped the vars into element & modifier vars, too.